cmd/object-api-errors.go

```diff
@@ -628,6 +628,15 @@ func (e PanFSS3InvalidName) Error() string {
 	return "Invalid symbols in object/bucket name."
 }
 
+// PanFSInvalidBucketPath bucket does not exist
+type PanFSInvalidBucketPath struct {
```

Comment:
it may be appropriate to add the `Error` prefix to the struct name

Reply:
Yes, that seems to be reasonable. But I'm just following the naming convention provided by the minio for that error level layer - they defined all errors structures there without `Err` prefix